Reinforced.Typings.Tests/TypeNameEqualityComparer.cs

```diff
@@ -20,7 +20,7 @@ public bool Equals(RtTypeName x, RtTypeName y)
         private bool CompareSimple(RtSimpleTypeName x, RtSimpleTypeName y)
         {
             if (x.TypeName != y.TypeName) return false;
-            if (x.Prefix != y.Prefix) return false;
+            if ((x.HasPrefix || y.HasPrefix) && x.Prefix != y.Prefix) return false;
```

Comment:
This was breaking tests. The `Prefix` property is always checked for `NullOrEmpty` - so I've added a check here also.